client/app/api/course/Assessment/Question/index.ts

```diff
@@ -1,9 +1,11 @@
 import McqMrqAPI from './McqMrq';
 import ScribingQuestionAPI from './Scribing';
+import VoiceAPI from './Voice';
 
 const QuestionAPI = {
   mcqMrq: new McqMrqAPI(),
   scribing: new ScribingQuestionAPI(),
+  voice: new VoiceAPI(),
```

Comment:
nit: rename to VoiceResponsesAPI(). Similarly, rename `Voice.ts` to `VoiceResponses.ts` and everything in that file.

Reply:
done